fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -210,11 +210,20 @@ protected Response getContent(final String rangeValue,
                 prefer.getReturn().addResponseHeaders(servletResponse);
             }
         }
-        servletResponse.addHeader("Vary", "Accept, Range, Accept-Encoding, Accept-Language");
 
+        addVaryHeader(resource());
         return ok(outputStream).build();
     }
 
+
+    protected void addVaryHeader(final FedoraResource resource) {
```

Comment:
I would suggest moving the logic of this method into `ContentExposingResource.addResourceHttpHeaders`.
I believe that will handle both the GET and HEAD scenarios.